.github/workflows/build_osi-validation.yml

```diff
@@ -0,0 +1,60 @@
+name: Build osi-validation
+
+on:
+  workflow_call:
+
+jobs:
+  build_esmini:
+    name: Build osi-validation
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cache osi-validation
+        id: cache-osi-validation
+        uses: actions/cache@v3
+        with:
+          path: /tmp/osi-validation
+          key: ${{ runner.os }}-osi-validation
+
+      - name: Get osi-validation
+        if: steps.cache-osi-validation.outputs.cache-hit != 'true'
+        working-directory: /tmp
+        env:
+          GIT_CLONE_PROTECTION_ACTIVE: false
+        run: git clone https://github.com/OpenSimulationInterface/osi-validation.git
+
+      - name: Checkout develop Branch
+        if: steps.cache-osi-validation.outputs.cache-hit != 'true'
+        working-directory: /tmp/osi-validation
+        run: git checkout develop_traffic_participant_model
```

Comment:
This needs to be changed to the main branch, once https://github.com/OpenSimulationInterface/osi-validation/pull/76 and https://github.com/OpenSimulationInterface/osi-validation/pull/74 are merged in osi-validation.

Reply:
Everything else is ready for review @jdsika.